From 1d14104b17074f40dbff1911879a5abc0473b96d Mon Sep 17 00:00:00 2001
From: Nico Lintner <118087966+nlintn@users.noreply.github.com>
Date: Thu, 27 Nov 2025 23:16:15 +0100
Subject: [PATCH 2/2] disable save changes modal on database lock

---
 src/gui/DatabaseWidget.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/gui/DatabaseWidget.cpp b/src/gui/DatabaseWidget.cpp
index 2afde49f..931fcffa 100644
--- a/src/gui/DatabaseWidget.cpp
+++ b/src/gui/DatabaseWidget.cpp
@@ -2097,6 +2097,7 @@ bool DatabaseWidget::lock()
             }
         }
 
+        /*
         if (!saved) {
             QString msg;
             if (!m_db->metadata()->name().toHtmlEscaped().isEmpty()) {
@@ -2119,6 +2120,7 @@ bool DatabaseWidget::lock()
                 return false;
             }
         }
+        */
     } else if (m_db->hasNonDataChanges() && config()->get(Config::AutoSaveNonDataChanges).toBool()) {
         // Silently auto-save non-data changes, ignore errors
         QString errorMessage;
-- 
2.51.2

